src/Ol3/Controls/ElevationPath.js

```diff
@@ -48,6 +48,11 @@ define([
     * @param {Object} [options.stylesOptions.draw.finish = {}] - Line Style when finished drawing. Specified with an {@link https://openlayers.org/en/latest/apidoc/ol.style.Stroke.html ol.style.Stroke} object.
     * @param {Object} [options.elevationPathOptions = {}] - elevation path service options. See {@link http://ignf.github.io/geoportal-access-lib/latest/jsdoc/module-Services.html#~getAltitude Gp.Services.getAltitude()} for available options
     * @param {Object} [options.displayProfileOptions = {}] - profile options.
+    * @param {Boolean} [options.displayProfileOptions.greaterSlope = true] - display the greater slope into the graph
+    * @param {Boolean} [options.displayProfileOptions.meanSlope = true] -  display the mean slope into the graph
+    * @param {Boolean} [options.displayProfileOptions.ascendingElevation = true] -  display the ascending elevation into the graph
+    * @param {Boolean} [options.displayProfileOptions.descendingElevation = true] -  display the descending elevation into the graph
+    * @param {Boolean} [options.displayProfileOptions.currentSlope = true] -  display the profile into the graph
```

Comment:
Dernier détail pour moi : il faudrait aussi changer la description du coup , par exemple "display current slope value on profile mouseover" plutôt que "display the profile into the graph"